### DISCUSSION TITLE
Admin dashboard

### DISCUSSION THREAD

**Discussion by redalert99** [simple-login/app#1723] [Self-hosting question]
Hello,

can someone tell me where to find the admin dashboard ?

I set is_admin for my user in the database.

Thank you.

Red

**Comment by t2noob** [comment]
Its right next to the settings icon. try https://app.yourdomain.com/admin/user, see if you can reach it that way. Otherwise you might need to update to the latest app-ci from docker.

**Comment by redalert99** [comment]
thanks, that works. how can i delete user ? only via the database or somehow different?

**Comment by t2noob** [comment]
From The Admin section you should be able to disable to user.  To completely erase it you would probably have to delete it from the database, but I would suggest you just disable it because the alias' would probably get deleted also or become unusable. Not entirely sure but I would just disable it to be on the safe side. admin->user->with selected-> disable user.

**Comment by MuhammadNaeem42** [comment]
i will help you to setup this application 
contact me https://www.fiverr.com/s/9W3PNA

![simple](https://github.com/simple-login/app/assets/106573678/790bab96-8316-4825-be68-8efbddaf8f07)

**Comment by 3gmgithub** [comment]
When I go here I get this error:

redirected you too many times.
[Try deleting your cookies](https://support.google.com/chrome?p=rl_error&hl=en-US).
ERR_TOO_MANY_REDIRECTS

**Reply by 3gmgithub** [reply]
I am using Traefik, and the main user interface works fine, but I was just trying to access the admin side...

**Reply by 3gmgithub** [reply]
How would I disable from the database if this does not work...

**Reply by 3gmgithub** [reply]
Ok looks like the Admin flag was not set for my user... Had to run the following commands...

```bash
# Get user list
select id, is_admin, email, lifetime from users order by id;

# Update user ID 1 to admin
update users set is_admin = TRUE where id=1;
```

Now it works!
